test/runtests.jl

```diff
@@ -1,9 +1,13 @@
 using Test, Agents, Random
 
+@testset "all tests" begin
```

Comment:
This choice could lead to problems in CI Travis if the entirety of the test set takes more than 10 minutes. Because with this choice there are messages printed only when all tests finish (or there is an aerror). Therefore if all tests take more than 10 minutes, Travis will error.

What is the motivation behind it?

Reply:
The motivation is to see all test failures at once. If a test fails, other tests are still run.
Currently, tests don't take more than a minute.

Reply:
Okay then, but let's keep this in mind as we add more tests.